examples/testing/multi_frameworks_toolchain/.bazelrc

```diff
@@ -0,0 +1 @@
+build --enable_runfiles
```

Comment:
Can `--enable_runfiles` be put under `windows` config?

Reply:
done.